chapter_07/jhh/exercise4.pl

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/perl
+my @lines = <>;
+
+foreach (@lines) {
+  if (/[A-Z]/ & /[a-z]/) {
```

Comment:
We suspected the single ``&`` here was a typo, but watch out for it as it does a bitwise-and rather than an 'and'. Fortunately here it happened to work!

Reply:
Damn good spot. Yeah should have been &&